testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -187,7 +187,8 @@
     "GMX_TOP_BAD", # file with an #include that doesn't exist
     "ITP_no_endif", # file missing an #endif
     "PDB_CRYOEM_BOX", #Issue 2599
-    "PDB_CHECK_RIGHTHAND_PA" # for testing right handedness of principal_axes
+    "PDB_CHECK_RIGHTHAND_PA", # for testing right handedness of principal_axes
+    "FHIAIMS" # to test FHIAIMS coordinate files
```

Comment:
add a comma to make it easier for the next addition
```suggestion
    "FHIAIMS", # to test FHIAIMS coordinate files
```

Reply:
done